packages/block-editor/src/components/index.js

```diff
@@ -175,3 +175,5 @@ export { useBlockCommands } from './use-block-commands';
  * The following rename hint component can be removed in 6.4.
  */
 export { default as ReusableBlocksRenameHint } from './inserter/reusable-block-rename-hint';
+
+export * from './border-box-utils';
```

Comment:
Would the `packages/block-editor/src/utils` be a better folder instead of the `components` folder for these utils?